File: tlu_contact_be/src/main/java/com/mobile/group/tlu_contact_be/model/Staff.java
package com.mobile.group.tlu_contact_be.model;

import lombok.*;
import lombok.experimental.FieldDefaults;

import java.util.List;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
@FieldDefaults(level = AccessLevel.PRIVATE)
public class Staff {
    String staffId;
    String fullName;
    String position;
    String phone;
    String email;
    String photoBase64;
    List<String> departmentIds;
    String userID;
    Boolean deleted = false;
}

File: TLUContact/app/src/main/java/com/mobile/group/tlucontact/models/Staff.kt
package com.mobile.group.tlucontact.models

import android.os.Parcel
import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class Staff(
    val staffId: String,
    val fullName: String,
    val position: String,
    val phone: String,
    val email: String,
    val photoURL: String,
    val unit: String,
    val userID: String,
    val avatarResId: Int
) : Parcelable